extensions/big-data-cluster/src/bigDataCluster/dialog/bdcDashboardModel.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+'use strict';
+
+import { IBdcStatus, IStatus, IServiceStatus } from '../controller/clusterControllerApi';
+
+export class BdcDashboardModel {
+
+	private _clusterStatus: IBdcStatus;
+
+	constructor(public clusterName: string, url: string, username: string, password: string) {
+
+	}
+
+	public get clusterStatus(): IStatus {
+		return { healthStatus: 'Warning detected', state: 'ready' };
+	}
+
+	public get serviceStatus(): IServiceStatus[] {
+		return [
+			{ serviceName: 'SQL Server', status: { state: 'Ready', healthStatus: 'Warning' }, resources: undefined },
+			{ serviceName: 'HDFS', status: { state: 'Ready', healthStatus: 'Healthy' }, resources: undefined },
+			{ serviceName: 'Spark', status: { state: 'Ready', healthStatus: 'Healthy' }, resources: undefined },
+			{ serviceName: 'Control', status: { state: 'Ready', healthStatus: 'Healthy' }, resources: undefined },
+			{ serviceName: 'Gateway', status: { state: 'Ready', healthStatus: 'Healthy' }, resources: undefined },
+			{ serviceName: 'App', status: { state: 'Ready', healthStatus: 'Healthy' }, resources: undefined }
+		];
+	}
+
+	public get serviceEndpoints(): { serviceName: string, hyperlink?: string, isHyperlink: boolean, ipAddress?: string, port?: string }[] {
+		return [
+			{ serviceName: 'SQL Server Master Instance', ipAddress: '10.91.134.112', port: '31433', isHyperlink: false },
```

Comment:
So: we're moving from ipAddress + port to endpoint, see my new PR. You'll want to update but doesn't need to be done in this review. Just note what I've done in serviceEndpoints.ts in my review for reference.

Reply:
Yup, my local changes are already set up using the new API - this is just pretty old 😄